packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -102,19 +102,18 @@ async function runAndroid(_argv: Array<string>, config: Config, args: Flags) {
     } else {
       // result == 'not_running'
       logger.info('Starting JS server...');
-      try {
-        startServerInNewWindow(
-          args.port,
-          args.terminal,
-          config.reactNativePath,
-        );
-      } catch (error) {
+
+      // Using a Promises-style catch here so the process runs in the background - awaiting it will cause it to hang.
+      startServerInNewWindow(
```

Comment:
So the idea is to run this function and not wait till it finishes, but return with `buildAndRun` on line 118?